File: Java/2186. Minimum Number of Steps to Make Two Strings Anagram II.java
class Solution {
    public int minSteps(String s, String t) {
        int temp = 0, sc[] = new int[26], tc[] = new int[26];

        for(char i: s.toCharArray()) sc[(int)i - 97] += 1;
        for(char j: t.toCharArray()) tc[(int)j - 97] += 1;

        for(int k = 0; k < 26; k++){
            if(tc[k] != 0 && sc[k] != 0) temp += Math.min(sc[k], tc[k]) * 2;
        }

        return s.length() + t.length() - temp;
    }
}

File: Python/2186. Minimum Number of Steps to Make Two Strings Anagram II.py
class Solution:
    def minSteps(self, s: str, t: str) -> int:
        temp, sc, tc = 0, Counter(s), Counter(t)

        for i, x in tc.items():
            if i in sc: temp += min(x, sc[i]) * 2
        
        return len(s + t) - temp
